islandora-event-support/src/main/java/ca/islandora/alpaca/support/event/AS2AttachmentContent.java

```diff
@@ -27,6 +27,36 @@
  */
 public class AS2AttachmentContent {
 
+    /**
+     * @return Fedora uri
+     */
+    @JsonProperty(value = "fedora_uri")
```

Comment:
Do we need this now that it's 'target'?

Reply:
And the answer is... yes, we do.  Files need the full url of where they're going.  Target is the fedora base url only.